cdap-client/src/main/java/co/cask/cdap/client/config/ClientConfig.java

```diff
@@ -128,14 +128,14 @@ public URL resolveNamespacedURLV3(Id.Namespace namespace, String path) throws Ma
 
   public HttpRequestConfig getDefaultRequestConfig() {
     if (connectionConfig == null) {
-      throw new DisconnectedException();
+      throw new DisconnectedException(null);
```

Comment:
Why not having an overloaded constructor for `DisconnectedException` that takes no argument so that caller don't need to pass in null explicitly?